src/components/BookList.jsx

```diff
@@ -1,69 +1,56 @@
-import { useDispatch } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 import React from 'react';
-import PropTypes from 'prop-types';
 import { deleteBook } from '../redux/slices/books/bookSlice';
 
-function BookList({ books }) {
+function BookList() {
+  const {
+    books, isLoading, errorMsg,
+  } = useSelector((store) => store.book);
   const dispatch = useDispatch();
```

Comment:
- [ ] Right now your app is not working as expected and whenever I try to add a book or remove it, I need to refresh the [age to see the difference. You may need to change your approach in order to meet the requirements. Kindly consider importing the `getBooks` action creator here from booksSlice and then also consider importing useEffect hook and then calling that action creator in the useEffect hook.